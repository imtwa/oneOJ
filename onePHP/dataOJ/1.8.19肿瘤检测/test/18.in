74
85 95 41 29 39 21 48 13 83 81 55 41 16 67 56 29 83 81 60 81 94 20 32 44 36 64 18 77 20 29 23 37 44 11 62 46 61 76 64 54 95 99 76 40 73 22 37 65 51 46 82 63 61 66 19 51 41 77 30 60 81 75 91 89 94 18 28 83 69 71 66 37 17 16
37 36 76 53 74 51 51 95 80 14 65 78 53 97 92 86 94 53 78 23 25 32 90 68 39 27 88 58 19 28 61 83 94 27 92 28 84 41 15 19 75 64 63 23 51 23 17 38 17 78 23 21 42 28 78 23 43 74 31 18 91 57 97 18 17 29 34 56 76 15 22 16 93 91
35 38 98 48 32 34 53 95 16 76 85 85 73 42 95 18 14 26 72 56 79 63 26 88 77 35 95 84 60 51 87 90 30 95 44 99 20 78 22 81 87 20 70 85 16 34 47 25 12 97 92 90 77 57 80 79 67 65 60 100 40 53 96 65 14 69 75 18 75 86 65 96 80 17
37 10 12 80 29 78 36 56 80 35 92 25 11 94 96 35 96 69 67 98 12 45 29 28 23 32 60 85 85 54 79 29 26 91 74 19 79 28 17 79 85 69 16 48 33 55 17 43 52 60 22 56 19 87 40 81 31 85 88 66 79 65 80 78 94 54 75 28 87 54 22 51 12 36
44 64 12 56 76 90 19 64 16 55 35 86 38 74 24 76 67 39 84 81 74 78 37 81 45 84 22 24 86 76 45 13 91 59 20 57 10 49 22 77 89 41 36 96 65 94 38 67 12 11 77 93 47 36 33 62 69 39 44 25 71 53 66 51 69 24 20 13 95 96 38 48 52 50
19 26 36 31 74 41 52 95 87 24 87 77 69 53 92 95 67 62 67 42 15 40 25 39 74 10 37 75 96 54 22 62 93 70 38 55 44 83 10 77 49 17 35 29 66 93 41 60 46 59 68 24 67 15 81 32 11 18 82 31 59 86 70 87 43 98 11 18 25 44 19 59 73 26
40 82 19 80 44 20 70 18 65 77 66 50 53 28 43 63 13 72 85 47 64 88 88 64 12 75 67 32 66 45 10 22 58 80 28 10 43 99 53 13 97 82 80 32 83 13 68 97 17 13 99 30 25 83 39 76 89 28 18 90 72 43 36 17 87 30 76 25 48 52 44 91 72 100
51 31 32 36 32 64 65 15 42 95 54 83 23 21 95 12 79 60 77 98 25 31 24 37 10 34 37 19 51 80 28 81 27 86 24 63 30 57 76 33 12 94 59 47 10 34 81 94 64 74 77 93 11 55 23 29 93 45 59 86 67 57 98 31 82 10 13 22 85 76 33 23 39 46
59 34 15 87 44 51 42 23 77 63 69 36 58 18 94 83 75 33 53 27 80 29 78 46 54 96 68 39 91 99 44 16 19 93 35 66 14 43 41 42 42 66 60 86 35 22 97 78 68 35 65 72 12 89 11 71 59 42 80 47 62 86 13 89 46 59 56 92 47 83 63 60 13 76
96 68 96 49 71 89 79 72 36 42 27 13 68 50 37 94 12 20 99 79 39 16 73 69 97 78 94 59 27 94 51 72 42 88 55 28 67 22 67 46 78 89 49 42 63 14 31 92 61 84 99 75 28 45 11 73 43 56 40 17 21 55 36 53 50 43 44 23 36 38 12 71 98 99
79 61 64 32 40 89 83 32 91 30 68 30 31 54 70 63 34 14 84 26 27 74 84 79 49 72 36 32 43 25 69 30 25 33 17 38 100 29 86 64 15 50 89 35 42 88 13 86 68 27 67 59 21 87 61 41 30 65 45 73 98 100 24 78 29 85 71 36 76 57 58 86 59 10
22 43 36 53 54 82 81 31 68 91 24 22 49 34 85 33 21 60 13 16 41 18 90 55 32 53 71 61 80 54 59 49 28 77 98 55 50 52 54 65 70 95 58 66 67 41 25 36 55 88 43 19 50 83 49 55 27 38 78 86 44 45 46 16 49 81 25 78 38 80 85 93 40 87
31 19 74 24 74 36 57 25 71 58 68 34 95 77 55 100 53 95 52 88 24 30 44 62 57 24 85 34 77 21 56 94 23 14 10 87 51 24 40 37 14 10 36 82 91 38 23 98 65 82 31 51 47 96 78 48 87 40 49 43 53 16 71 42 98 52 47 70 67 37 79 53 58 60
74 48 61 40 55 78 19 93 46 30 64 49 71 11 72 20 65 33 78 26 40 15 65 56 79 57 61 42 83 97 73 45 13 25 89 29 81 86 75 24 74 92 69 65 91 88 98 50 98 40 40 72 63 31 28 78 67 73 22 72 80 69 81 20 76 49 17 89 10 57 83 19 51 19
85 44 83 85 94 36 36 16 78 94 51 35 50 62 71 66 26 99 61 24 54 14 63 22 90 77 16 88 42 95 45 60 88 77 20 27 47 78 23 31 86 15 24 76 82 86 35 65 16 38 10 96 55 20 53 18 54 93 27 58 36 19 96 97 35 73 25 97 18 25 83 24 56 15
59 81 84 11 40 52 14 33 28 94 49 35 65 73 40 70 60 23 77 34 78 49 22 14 68 94 11 79 82 53 12 64 21 14 33 61 77 82 45 99 72 98 97 22 13 32 77 18 91 17 58 70 69 33 45 30 52 97 10 17 27 63 18 57 42 94 22 86 97 50 52 78 95 11
55 39 79 32 21 16 19 22 97 74 59 90 35 60 62 39 26 59 59 26 49 40 46 10 77 97 78 10 74 79 91 95 89 49 43 19 99 11 32 14 98 31 85 68 62 97 78 75 58 89 19 65 12 80 95 83 74 13 50 57 82 33 41 20 94 82 83 22 57 30 73 50 14 11
78 25 88 46 33 33 48 14 36 98 76 24 11 62 100 65 90 26 10 29 19 73 70 30 67 38 88 35 87 93 66 38 72 24 46 41 51 88 57 93 40 32 30 38 64 87 87 76 49 52 16 93 90 29 25 74 12 92 18 34 48 16 99 48 100 97 52 31 56 70 20 11 55 33
31 95 33 45 59 69 39 21 70 20 78 52 49 16 68 90 14 24 68 55 21 32 32 21 33 12 46 54 18 40 53 92 86 50 41 53 95 41 80 89 80 81 75 99 34 64 23 66 19 100 89 75 14 36 66 14 94 85 81 74 57 36 69 97 75 20 15 34 84 24 21 85 43 78
90 46 20 99 90 85 33 60 55 89 81 88 39 81 10 21 74 50 68 99 58 83 59 53 35 100 29 40 23 37 33 23 78 65 56 78 27 38 94 26 44 56 60 69 48 60 27 17 18 48 100 28 74 97 87 70 59 93 13 17 63 74 78 62 67 98 18 24 61 62 72 31 30 55
49 65 68 56 52 38 63 14 85 67 81 16 12 28 27 26 15 39 59 60 56 86 91 66 27 25 69 44 31 64 93 41 93 98 90 36 66 20 25 12 25 24 18 22 46 23 71 20 71 20 43 43 91 85 75 35 80 24 81 12 46 56 17 48 14 63 33 77 14 57 61 30 53 35
78 21 47 23 88 76 78 76 13 97 23 69 99 36 77 87 87 10 48 23 74 46 34 16 25 54 87 46 49 41 28 69 51 32 63 55 29 43 36 84 27 65 15 30 16 25 75 72 98 38 60 37 77 36 10 62 100 95 23 55 74 83 53 37 44 20 47 71 35 70 53 17 52 26
21 18 66 70 75 85 96 97 13 94 60 26 97 73 71 24 25 74 27 18 82 76 66 62 33 44 40 53 99 80 41 76 33 81 53 30 15 63 16 29 36 58 91 62 86 86 31 56 93 10 15 69 97 44 17 43 15 32 35 75 58 85 64 62 73 57 42 77 29 59 100 45 11 78
46 80 76 36 66 44 88 25 35 18 58 56 47 10 10 64 91 23 38 36 87 66 64 45 70 47 21 35 15 49 17 29 32 25 85 71 13 15 18 64 25 42 15 81 10 59 93 51 69 43 82 21 36 68 19 100 48 78 91 71 54 47 70 45 31 65 55 76 69 61 92 31 42 93
46 40 71 69 67 68 20 53 51 54 53 66 61 30 23 77 80 78 98 46 34 43 95 81 26 97 91 55 97 43 24 58 43 48 56 25 92 55 60 91 26 98 14 85 29 56 56 55 58 72 12 11 24 40 100 83 80 77 57 23 11 56 53 78 46 36 24 14 42 14 54 15 79 82
55 36 94 54 61 24 50 76 21 10 78 38 95 26 81 86 90 57 39 36 34 42 25 41 34 24 57 92 48 58 88 99 52 81 74 29 52 72 66 72 52 20 17 24 58 10 68 71 55 58 65 100 96 32 73 41 72 98 54 55 75 34 30 81 16 59 92 38 45 93 15 14 51 95
15 61 91 66 22 94 57 59 100 49 58 81 93 72 62 30 82 62 37 43 30 87 67 15 11 14 43 40 12 54 60 24 33 58 22 62 36 26 33 27 23 17 31 78 35 61 85 48 99 12 72 52 69 48 17 39 92 16 21 82 83 66 20 91 82 11 94 17 24 18 14 58 89 44
27 58 90 93 64 72 91 22 86 54 39 15 30 57 73 61 31 11 47 13 60 67 61 88 25 59 38 30 40 14 46 97 41 97 75 54 82 37 54 26 47 77 98 36 43 55 28 38 65 78 89 52 10 96 85 43 95 70 51 89 40 82 29 44 45 10 88 42 50 98 10 60 45 76
57 96 27 27 12 14 25 72 89 22 35 82 34 97 68 19 20 97 17 67 18 24 12 93 66 76 99 84 97 31 29 97 66 30 46 68 88 44 79 88 21 52 73 30 19 86 38 71 70 19 100 58 77 50 87 54 16 57 74 81 24 36 30 59 60 35 29 48 34 47 28 99 65 31
64 63 22 88 34 46 17 69 72 14 19 10 64 72 66 74 19 16 74 100 27 22 20 47 91 88 97 67 61 32 26 35 91 30 34 70 86 98 95 85 73 39 26 37 24 54 56 47 69 45 88 98 61 14 68 44 26 54 64 29 78 63 88 13 33 67 60 47 74 82 74 11 55 28
88 51 81 99 82 78 88 54 48 96 50 58 67 42 34 32 59 85 98 89 78 51 45 94 87 35 83 94 59 49 53 36 34 71 73 93 97 68 85 98 69 64 13 60 88 94 53 32 12 89 25 47 69 17 90 25 31 41 17 27 26 48 44 23 12 59 58 18 43 27 41 25 48 79
86 100 26 52 95 76 38 83 24 48 50 84 52 14 81 97 45 19 98 72 83 66 71 86 40 38 50 60 86 31 41 63 46 46 38 59 26 32 58 61 56 93 12 55 30 88 73 28 45 76 42 47 56 31 45 79 35 99 84 64 67 30 54 61 87 95 10 42 97 25 69 23 61 46
51 96 93 93 60 49 47 76 56 55 31 96 86 78 25 38 31 13 100 53 57 93 86 50 77 100 76 39 31 63 38 71 79 74 29 41 18 25 56 15 69 34 97 75 14 45 12 66 36 54 29 32 66 57 14 39 52 77 74 38 46 72 53 38 71 64 100 60 28 48 43 13 21 30
67 88 70 10 19 90 94 21 61 77 86 91 83 56 70 99 76 59 17 64 59 96 70 12 35 39 94 84 67 19 78 79 36 24 17 95 24 100 39 23 86 59 14 74 15 62 76 31 83 28 83 16 71 28 12 42 23 67 17 50 64 93 61 37 17 78 63 51 61 41 52 23 25 48
62 65 37 59 100 31 60 26 84 56 28 74 26 53 97 35 44 16 75 69 16 26 42 31 90 99 79 63 62 17 74 18 63 48 92 80 30 24 91 15 60 36 82 77 26 55 56 68 76 75 100 66 57 58 85 35 88 61 47 39 75 96 41 68 33 84 63 18 97 98 95 94 99 44
54 79 40 49 27 44 14 82 66 92 67 82 85 88 80 37 37 35 62 41 21 92 39 45 77 41 100 38 92 32 94 85 12 49 26 92 75 89 28 51 50 43 21 50 48 28 68 31 64 100 43 59 78 67 84 60 72 73 93 26 48 45 60 16 46 28 20 47 74 55 87 82 74 27
15 31 79 69 48 91 49 86 80 100 22 59 64 45 95 19 67 63 41 62 50 12 78 21 95 41 45 61 37 29 17 60 59 70 46 92 76 40 50 13 71 25 61 10 62 16 33 51 32 57 21 66 39 75 75 25 97 89 83 66 71 78 54 45 50 70 82 56 63 33 71 71 39 73
31 21 85 32 33 35 53 27 82 65 38 36 78 33 24 11 24 92 57 64 53 54 28 48 12 15 15 57 14 69 73 62 11 84 28 59 11 97 61 34 21 29 38 82 78 38 12 62 49 88 42 90 10 55 10 64 80 34 10 45 19 87 30 67 89 52 92 55 49 55 13 89 20 95
81 71 24 36 48 23 46 23 96 95 19 18 17 37 36 60 57 24 71 25 67 64 32 96 81 55 55 11 20 68 74 59 24 13 16 70 30 87 15 98 22 90 46 82 34 66 20 20 30 52 55 56 14 30 82 67 55 61 30 54 66 38 88 25 31 91 91 74 88 41 92 10 83 19
92 62 90 69 78 86 40 91 47 60 26 55 37 63 40 80 75 32 35 25 74 67 33 35 71 43 47 66 25 29 61 54 10 91 56 17 58 92 46 34 77 71 46 46 16 40 30 19 85 11 77 61 22 75 60 74 39 69 98 36 62 94 16 56 59 95 28 47 30 70 14 27 39 85
95 99 69 93 63 77 39 62 16 55 97 49 80 26 51 56 20 94 60 36 80 54 62 57 83 44 98 20 61 61 48 69 63 92 82 63 91 30 11 64 21 40 74 69 96 56 45 15 50 20 90 21 93 28 16 19 76 35 79 48 76 64 18 81 53 10 67 18 22 42 16 31 29 13
89 80 42 47 17 40 12 42 10 19 43 85 40 40 82 11 30 32 42 55 78 43 69 96 36 21 87 79 53 12 82 21 79 57 91 76 15 19 28 12 57 68 88 30 21 46 51 60 60 31 93 23 36 74 73 31 16 44 26 17 39 54 42 37 75 81 40 79 26 66 99 84 14 94
86 51 53 59 28 66 54 50 87 69 63 51 23 17 73 12 50 22 53 53 20 35 44 73 38 35 39 99 90 34 61 15 74 29 78 44 29 57 28 74 52 57 90 37 64 35 63 59 11 26 53 83 64 82 93 96 52 33 64 82 74 12 62 21 30 39 97 73 97 25 99 89 81 54
77 36 52 26 72 93 87 27 82 94 44 25 92 38 72 38 78 67 63 46 20 41 21 53 96 31 29 44 79 84 58 51 50 69 72 47 56 45 31 59 97 91 70 52 89 40 19 43 15 52 90 68 50 31 78 57 47 97 55 34 51 93 61 50 30 76 27 94 66 42 56 85 12 17
84 60 70 28 78 25 39 32 70 95 96 95 61 38 25 97 63 41 88 28 87 68 63 93 14 24 56 55 95 27 81 39 49 89 99 67 72 87 70 85 43 37 65 17 26 11 16 72 23 96 16 60 49 41 30 100 14 99 59 28 56 15 96 85 24 41 10 46 73 20 15 80 73 86
28 63 78 92 81 58 78 34 82 14 98 29 95 44 48 98 40 42 57 82 87 90 58 76 74 30 34 72 85 99 35 85 23 11 68 89 48 42 31 90 93 67 52 53 83 98 67 97 84 82 92 91 43 14 22 19 84 28 50 54 11 13 80 14 66 40 41 87 16 92 52 30 66 60
42 86 98 48 69 10 46 17 60 57 41 61 26 48 32 46 87 33 76 67 52 28 35 40 50 65 70 49 84 39 64 58 91 91 45 83 37 74 24 39 65 31 98 30 97 48 52 25 24 32 37 12 49 49 99 63 82 77 19 57 46 98 99 85 51 93 16 99 80 66 27 39 51 26
50 96 71 76 41 32 67 91 93 11 89 91 78 58 14 32 89 65 34 63 68 51 43 90 23 91 73 89 27 44 44 66 63 71 32 15 85 65 27 78 51 54 94 92 20 24 50 93 37 43 58 95 34 50 69 87 46 18 86 26 97 56 31 97 64 17 40 29 33 21 78 25 89 41
68 66 99 96 89 59 83 53 31 13 68 94 98 55 23 87 65 50 63 72 57 77 41 63 86 24 79 60 80 85 43 76 90 87 79 71 25 34 82 73 79 70 65 11 68 35 69 74 78 33 94 93 61 94 26 80 97 58 12 63 84 81 59 20 38 68 85 54 86 94 50 81 19 78
14 56 12 74 15 83 13 99 12 47 94 86 93 53 77 60 79 23 30 100 88 31 57 29 93 54 54 89 68 10 10 26 91 32 56 84 93 92 22 96 29 83 65 85 68 41 78 28 33 37 85 13 93 43 29 23 78 83 52 55 69 40 21 91 43 11 59 98 35 99 20 81 99 93
55 89 98 77 24 32 67 91 27 13 20 98 85 78 29 69 89 96 76 34 19 88 32 82 57 48 60 47 93 90 92 28 12 47 91 10 28 74 20 11 49 45 25 99 16 39 26 21 56 44 20 97 76 30 90 45 44 18 64 58 48 82 43 18 74 42 73 50 96 81 88 97 18 32
34 19 68 25 24 20 46 16 53 14 78 25 100 91 48 13 97 84 30 99 41 69 14 99 33 18 99 69 75 25 26 28 86 76 59 72 95 45 41 13 76 30 58 32 22 86 20 74 98 13 44 66 28 43 97 69 57 32 71 10 23 48 30 87 48 24 73 33 10 59 30 51 96 25
82 23 23 97 98 90 83 25 42 47 90 33 52 38 65 63 88 49 41 83 22 57 50 53 18 28 63 93 49 65 55 30 62 28 88 59 87 89 10 74 30 36 44 55 51 42 35 71 73 55 34 58 18 13 19 37 14 63 60 12 95 49 74 28 12 60 55 61 48 50 80 10 70 98
45 25 94 44 85 80 38 37 85 84 84 80 57 60 46 75 83 73 83 97 84 26 76 44 11 15 28 12 26 41 60 55 28 27 78 73 10 53 54 50 14 46 24 23 57 20 27 28 29 40 14 39 57 45 17 90 41 13 57 96 16 55 87 53 36 54 48 36 50 68 10 21 68 81
12 15 69 46 45 64 93 52 29 97 93 37 57 14 24 88 63 65 11 28 43 79 76 32 13 83 10 44 61 98 64 25 72 14 66 58 66 49 98 43 39 40 56 53 44 26 22 33 25 93 14 65 95 86 29 62 95 59 17 25 78 43 32 81 78 46 96 58 29 60 59 12 92 99
19 95 36 18 39 97 89 99 19 51 97 75 48 21 84 58 48 20 13 59 76 17 41 67 36 60 57 74 17 69 95 23 66 89 17 72 24 73 27 45 21 42 68 42 35 38 24 18 68 86 19 22 79 49 79 100 17 17 37 99 90 39 71 37 71 12 50 93 27 77 10 31 37 39
68 40 22 54 40 78 35 14 90 20 49 15 96 35 40 56 100 44 76 77 97 36 49 80 20 95 59 91 14 89 46 79 62 18 79 51 37 83 85 72 63 71 67 26 15 69 11 98 14 77 92 19 51 78 34 70 66 10 93 58 98 64 48 42 74 68 21 40 52 93 71 13 14 68
32 51 39 64 68 74 93 69 87 98 92 87 70 42 14 86 76 76 80 41 10 18 90 17 65 63 27 37 90 10 84 61 60 21 29 77 22 97 83 29 38 75 39 18 56 81 98 84 37 92 20 45 32 57 21 86 27 24 29 81 35 41 66 67 71 24 63 31 22 22 91 38 68 28
41 20 83 74 72 84 36 53 73 77 90 14 74 21 15 67 93 42 33 16 19 82 74 70 38 22 16 10 73 69 61 76 84 87 87 61 58 15 60 60 99 46 18 16 56 31 61 62 10 25 21 74 98 23 57 11 100 100 77 46 59 96 51 83 60 85 28 86 10 53 14 44 12 60
38 27 22 69 23 35 13 51 31 92 23 59 56 33 66 46 51 25 67 27 68 49 26 23 78 55 29 27 16 92 82 72 30 51 36 38 93 64 41 24 63 33 72 58 39 92 95 56 84 44 89 84 89 95 32 42 18 41 42 57 73 82 53 54 41 45 12 41 28 10 14 75 19 97
62 12 44 16 33 46 38 83 41 78 24 85 12 26 67 38 95 75 47 75 98 94 48 47 38 25 42 76 59 53 95 35 87 46 17 39 11 10 70 15 41 67 85 42 72 93 33 73 13 58 14 90 19 49 14 51 99 52 46 97 64 80 24 50 69 40 63 32 14 28 31 55 99 84
27 38 75 47 14 70 17 62 72 38 74 75 35 97 74 89 34 24 36 61 61 68 28 44 86 47 58 59 30 75 58 26 33 32 44 29 62 15 35 54 15 59 60 79 89 29 82 65 25 64 32 85 10 64 49 16 10 87 19 73 94 36 71 20 21 56 34 85 82 49 47 73 80 18
51 15 11 80 76 48 27 30 21 62 39 20 79 50 19 87 27 39 23 10 41 27 87 56 77 39 20 65 80 87 10 78 33 50 35 15 95 68 84 42 44 26 34 64 65 66 17 68 48 55 88 30 33 62 24 21 31 38 69 11 38 99 72 21 85 66 90 37 91 27 85 15 38 37
80 54 45 45 63 62 55 74 31 20 16 64 55 41 38 48 11 63 95 30 35 84 18 46 69 52 68 60 78 88 67 64 22 55 83 29 44 67 92 34 61 71 73 55 29 39 29 60 77 16 44 33 60 54 65 83 36 49 89 41 12 49 14 19 63 49 18 46 39 43 50 72 34 32
12 35 93 100 46 78 41 18 10 26 47 52 71 76 78 29 46 16 74 80 23 72 95 68 59 33 73 14 75 43 96 41 19 28 71 64 39 34 98 94 29 48 55 94 23 70 48 69 61 42 65 38 88 72 55 87 67 37 39 30 67 88 19 17 93 98 60 98 95 91 70 40 34 91
72 93 11 37 12 25 41 64 46 23 37 46 25 34 52 10 82 12 36 85 42 88 28 98 14 12 64 90 12 30 98 30 80 75 78 12 94 19 95 87 46 72 28 19 20 78 97 29 55 65 65 34 30 10 11 43 19 75 39 22 50 94 60 65 21 68 60 42 79 95 86 54 21 88
67 18 22 82 28 50 28 11 77 63 83 58 63 83 94 49 30 77 14 93 55 62 55 37 28 66 59 71 27 58 83 28 99 31 35 82 38 70 92 35 25 55 92 15 92 36 43 17 20 29 11 83 87 27 56 14 35 26 75 28 54 26 11 84 87 99 56 84 75 87 82 55 48 95
77 11 10 58 44 29 30 50 28 93 23 41 20 24 34 35 46 53 21 56 44 96 32 83 59 34 60 36 55 64 12 18 53 43 12 25 19 51 68 97 69 44 99 73 18 80 87 54 77 77 20 49 21 30 78 61 22 43 15 77 74 66 87 100 69 58 14 68 33 93 23 62 64 41
49 78 16 37 48 57 28 18 87 37 51 74 78 73 85 33 12 85 34 93 86 68 97 88 11 62 89 68 69 22 94 70 74 55 89 22 80 75 58 43 21 23 65 100 46 51 77 10 81 21 62 10 27 88 57 39 83 77 21 100 38 12 83 96 97 25 15 59 10 98 22 43 94 33
58 64 90 61 46 42 76 22 52 86 95 99 80 67 51 66 84 56 75 62 22 42 17 41 53 82 21 24 42 12 53 85 70 29 65 36 43 18 61 39 35 49 14 77 50 39 50 34 86 34 29 35 22 81 32 38 48 85 25 62 69 30 27 52 74 97 64 82 18 95 21 31 99 71
36 92 76 64 34 33 83 17 31 12 90 52 19 52 51 43 26 99 85 38 53 73 55 41 95 39 96 66 52 87 61 21 39 49 47 31 93 30 43 23 59 23 14 92 76 59 96 40 20 78 17 93 62 49 55 67 96 64 63 29 81 16 79 68 31 89 50 23 97 12 33 94 97 54
35 40 65 61 30 61 62 45 33 18 96 99 77 74 97 100 18 98 70 16 14 12 79 10 25 69 84 70 72 73 97 88 52 11 30 93 55 30 97 83 56 85 12 27 41 31 66 77 50 25 86 82 47 89 63 50 19 34 94 24 70 18 59 44 50 33 50 99 91 82 67 55 82 47
59 98 86 21 34 41 81 39 68 96 79 13 80 45 73 22 72 82 33 69 45 31 24 45 14 49 53 11 96 58 74 69 58 81 68 65 51 50 34 13 63 45 35 45 70 30 70 72 99 34 23 52 21 70 36 81 19 32 32 39 88 24 90 48 97 14 80 90 63 51 93 71 70 38
33 19 35 48 98 19 27 13 65 78 89 42 26 82 38 98 77 81 57 86 83 89 18 31 53 58 79 33 71 49 46 29 33 96 22 62 92 38 94 55 89 90 39 90 40 94 18 22 58 87 82 70 100 66 95 25 83 62 31 50 53 91 99 84 86 19 66 69 69 93 79 98 24 71
