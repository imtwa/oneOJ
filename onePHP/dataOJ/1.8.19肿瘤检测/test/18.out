2536 2430
